packages/shared/src/globalsAllowList.ts

```diff
@@ -1,9 +1,30 @@
 import { makeMap } from './makeMap'
 
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects
 const GLOBALS_ALLOWED =
-  'Infinity,undefined,NaN,isFinite,isNaN,parseFloat,parseInt,decodeURI,' +
-  'decodeURIComponent,encodeURI,encodeURIComponent,Math,Number,Date,Array,' +
-  'Object,Boolean,String,RegExp,Map,Set,JSON,Intl,BigInt,console'
+  // Simple Global Values
+  'Infinity,NaN,undefined,' +
```

Comment:
I think the plain string is enough, instead of concating. Also, we think only `Error` (the only one), and `Symbol` can be included.